hw7/script.js

```diff
@@ -0,0 +1,159 @@
+const checkString = function (string) {
+  let result = true;
+  if (typeof string !== "string") {
+    result = false;
+  }
+  return result;
```

Comment:
Much shorter version 

```suggestion
return value.length && typeof value === "string"  || false
```

Reply:
or even 1-line function 
`const checkString =  (string) => value.length && typeof value === "string" || false`